### DISCUSSION TITLE
! LaTeX Error: File `footnotebackref.sty' not found.

### DISCUSSION THREAD

**Discussion by tobezhanabi** [Cyfrin/security-and-auditing-full-course-s23#116] [Q&A]
```
Error producing PDF.
! LaTeX Error: File `footnotebackref.sty' not found.
```
ive tried some uninstallation and installation and still landed o this error, couldnt find a good resource

**Comment by alymurtazamemon** [comment]
@tobezhanabi I have not reached this part, but I found a similar issue which might help you. Here you can find [this](https://github.com/Wandmalfarbe/pandoc-latex-template/issues/141)

**Reply by tobezhanabi** [reply]
yea, thanks i resolved it
